shared/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "replicache-quickstarts-shared",
+    "version": "0.1.0",
+    "private": true,
+    "devDependencies": {
+      "@rocicorp/eslint-config": "^0.1.2",
+      "@rocicorp/prettier-config": "^0.1.1",
+      "npm-run-all": "^4.1.5",
+      "typescript": "4.7.4"
+    },
+    "scripts": {
+      "format": "prettier --write './**/src/**/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
+      "check-format": "prettier --check './**/src/*/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
+      "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
+      "build": "tsc"
+    },
+    "type": "module",
+    "eslintConfig": {
+      "extends": "@rocicorp/eslint-config"
+    },
+    "prettier": "@rocicorp/prettier-config",
+    "engines": {
+      "node": ">=16.15.0",
+      "npm": ">=7.0.0"
+    },
+    "main": "src/index.ts",
+    "module": "src/index.ts"
```

Comment:
remove "module", from the docs it doesn't seem to be a valid field, "main" is the right thing (or for more complicated thing "exports").